tests/bsim/bluetooth/host/att/sequential/dut/prj.conf

```diff
@@ -0,0 +1,41 @@
+CONFIG_LOG=y
+CONFIG_ASSERT=y
+
+CONFIG_BT=y
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_DEVICE_NAME="Sequential"
+
+CONFIG_BT_GATT_CLIENT=y
+CONFIG_BT_GATT_ENFORCE_SUBSCRIPTION=n
+
+# Enable bt_testing_tx_tid_get()
+CONFIG_BT_TESTING=y
+
+# Replace `Execute` with `gdb --args` in the shell script
+# and get a nice backtrace on assert
+CONFIG_ARCH_POSIX_TRAP_ON_FATAL=y
+
+# Prepend logs with thread names
+CONFIG_THREAD_NAME=y
+CONFIG_LOG_THREAD_ID_PREFIX=y
+
+# Enable those as needed
+# CONFIG_BT_L2CAP_LOG_LEVEL_DBG=y
+# CONFIG_BT_ATT_LOG_LEVEL_DBG=y
+# CONFIG_BT_GATT_LOG_LEVEL_DBG=y
```

Comment:
Wouldn't it make sense for the runner of this test to add these themselves by need, rather than always keeping them in here?

Reply:
Yes, but it's always a pain to remember the exact name. Now they're easily accessible.

Reply:
> Yes, but it's always a pain to remember the exact name. Now they're easily accessible.

I guess that depend on the developer and their own development environment and IDE setup ;) 

Not a huge deal anyways